config/crd/bases/oadp.openshift.io_dataprotectionapplications.yaml

```diff
@@ -445,6 +445,24 @@ spec:
                 description: Features defines the configuration for the DPA to enable
                   the OADP tech preview features
                 properties:
+                  dataMoverCredential:
```

Comment:
I'm wondering should we move these configs to be:
```
features:
  datamover:
    enable: true
    credentials: <secret>
```

This way if we have more configs we can stick it under a top level `datamover` key?